modules/apps/layout/layout-page-template-api/src/main/resources/com/liferay/layout/page/template/validator/dependencies/page_definition_json_schema.json

```diff
@@ -3,10 +3,68 @@
 	"$schema": "http://json-schema.org/draft-07/schema#",
 	"additionalProperties": false,
 	"definitions": {
+		"ClassNameReference": {
+			"additionalProperties": false,
+			"properties": {
+				"className": {
+					"title": "The className Schema",
+					"type": "string"
+				}
+			},
+			"required": [
+				"className"
+			],
+			"title": "The ClassNameReference Schema",
+			"type": "object"
+		},
+		"ClassPKItemReference": {
```

Comment:
I think it would be better to name this ClassPKReference which is more consistent with ClassNameReference. Also, since it's used to reference a Collection too, it's weird that it has the term "Item" in its name.

Reply:
Makes sense. I'll rename it to ClassPKReference.

Reply:
Being ClassPKReference the name of the definition, I think it makes sense to rename the properties itemClassName and itemClassPK to className and classPK

Reply:
We also have ContextItemReference.
I'm inclined to rename it too to ContextReference. Any objections?

Reply:
Yes, it makes sense to change the properties inside the Class*Reference types too.

Regarding ContextItemReference, I had more doubts about whether ContextReference is too generic, but at the end I agree with you that it's better.